src/custom/utils/operator/index.ts

```diff
@@ -185,53 +191,79 @@ function toApiAddress(address: string, chainId: ChainId): string {
   return address
 }
 
-async function _getJson(chainId: ChainId, url: string): Promise<any> {
-  let response: Response | undefined
-  let json
-  try {
-    response = await _get(chainId, url)
-    json = await response.json()
-  } finally {
-    if (!response || !json) {
-      throw new Error(`Error getting query @ ${url}`)
-    } else if (!response.ok) {
-      // is backend error handled at this point
-      const errorResponse: ApiError = json
-      throw new OperatorError(errorResponse)
-    } else {
-      return json
-    }
+const UNHANDLED_QUOTE_ERROR: QuoteErrorObject = {
+  errorType: QuoteErrorCodes.UNHANDLED_ERROR,
+  description: QuoteErrorDetails.UNHANDLED_ERROR
+}
+
+const UNHANDLED_ORDER_ERROR: ApiErrorObject = {
+  errorType: ApiErrorCodes.UNHANDLED_CREATE_ERROR,
+  description: ApiErrorCodeDetails.UNHANDLED_CREATE_ERROR
+}
```

Comment:
This might be a problem, because in my changes, I've split this into `create` and `delete` errors, and here this is used on `get` errors.
Probably my change was not a good because I forgot the `get` case :sweat: 

Anyway, the message displayed will be `'The order was not accepted by the network'`

Reply:
well it was this before. i can create an `UNHANDLED_GET_ERROR` that's: `"Order fetch failed. This may be due to a server error or network connectivity issue. Please try again later."`

Reply:
9f56130